### DISCUSSION TITLE
Mem Inspector Enhancement: Save/Restore Search results

### DISCUSSION THREAD

**Discussion by kdecks** [RetroAchievements/RAIntegration#775] [Completed]
As a developer I would like to be able to save and then later restore my current search results, and possibly the results history, if that's available.

This would aide in the development process for the times that you need to turn off your emulator, but you've worked hard for the results you have, or for times when you'd like to search for something else, but still come back to the previous results for later refinement or study. It also may be nice in some cases to share the file with other developers.

### A mockup of how it could be presented in the Memory Inspector
![image](https://user-images.githubusercontent.com/32706333/48313882-5be2d600-e57f-11e8-99dd-5014e280a8cb.png)

**Comment by Jamiras** [comment]
I can see the potential value for a "copy to clipboard" functionality for sharing search results, but I'm less convinced about the functionality related to storing the state across sessions. Filter results are tied to a specific frame of emulation/save state. If you load those results against another frame/save state, will they still be valid? I suppose they could serve as an initial state for the next click of the Filter button - will have to think about this some more.

**Comment by ghost** [comment]
This could be useful when searching together as team for hard to find adresses/values so we could share the savestate files and the result history.

Maybe dropbox or any other sync software can be used to share those files automatically and syncronized in a group to increase the efficient of finding addresses/values.

**Comment by Jamiras** [comment]
Export was added by #926, and import was added by #1088.
